### PR TITLE
7.x 4.15 beta performance tweaks

### PR DESCRIPTION
**Pull request by epicflux**

### CODE REVIEW

File: fundraiser/modules/fundraiser_commerce/fundraiser_commerce.module [hunk 1]
```diff
@@ -1054,6 +1067,7 @@ function fundraiser_commerce_fundraiser_donation_create($donation) {
   $quantity = isset($donation->donation['quantity']) && !empty($donation->donation['quantity']) ? $donation->donation['quantity'] : 1;
 
   // Give other modules an opportunity to generate the order entity.
+  // @TODO nobody implements this. Can we get rid of it?
```
Comment (epicflux) [inline fundraiser/modules/fundraiser_commerce/fundraiser_commerce.module]: @pcave  Answering this TODO: this function is used in custom code on client sites. I think its best to remove or rewrite the TODO.

Comment (pcave) [inline fundraiser/modules/fundraiser_commerce/fundraiser_commerce.module]: Ok. Do we have to re-add the commerce_order_save() call as well? I think Scott figured out we save the order 8 times during a single request and was trying to eliminate any unnecessary calls.

Comment (scottatdrake) [inline fundraiser/modules/fundraiser_commerce/fundraiser_commerce.module]: There is a commerce_order_save() further down in this same function. Taking this order_save out didn't appear to have any impact but perhaps clients do doing custom things here that I'm not aware of, perhaps in the hook_ fundraiser_commerce_generate_line_items() that follows.

Comment (epicflux) [inline fundraiser/modules/fundraiser_commerce/fundraiser_commerce.module]: I haven't noticed any issues with removing the commerce_order_save().

**Comment by epicflux** [comment]
Closing this in favor of #1413

**Pull request closed without merging**